recipes/tassel/4.3.15/build.sh

```diff
@@ -0,0 +1,31 @@
+BINARY_HOME=$PREFIX/bin
+TASSEL_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+# Download TASSEL4 from its public repo
+git clone git://git.code.sf.net/p/tassel/tassel4-standalone tassel4-standalone
+cd tassel4-standalone
+git checkout V$PKG_VERSION
```

Comment:
Same as above: In `meta.yaml`:
```yaml
source:
  git_url: https://git.code.sf.net/p/tassel/tassel4-standalone.git
  git_rev: V{{ version }}
```

Reply:
Changed to download snapshot from SourceForge in the latest commit.